_layouts/page.html

```diff
@@ -66,6 +66,9 @@ <h1 class="post-title-main">{{ page.title }}</h1>
                     <a target="_blank" rel="noopener" href="{{site.github_editme_path}}{{page.path}}" class="btn btn-default githubEditButton" role="button"><i class="fab fa-github fa-lg"></i> Edit me</a>
                 </div>
             {% endif %}
+            <div class="text-right">
+              <small>Updated {{ page.last_modified_at | date:  '%d %b %y' -}}</small>
```

Comment:
How//where is this determined? Based on what?

Reply:
This plugin determines a page's last modified date by checking the last Git commit date of source files. In the event Git is not available, the file's mtime is used.

https://github.com/gjtorikian/jekyll-last-modified-at?tab=readme-ov-file#last-modified-at-plugin